nextflow.config

```
@@ -157,6 +157,11 @@ params {
     // profile standardisation
     run_profile_standardisation             = false
     standardisation_taxpasta_format         = 'tsv'
+    taxpasta_taxonomy_dir                   = false
```

```suggestion
    taxpasta_taxonomy_dir                   = null
```

I think? It's not a boolean, it's a path?

---

Yes, it is a path.